source/dubregistry/registry.d

```
@@ -399,6 +393,10 @@ class DubRegistry {
 			return false;
 		}
 
+		if ("description" !in info.info || "license" !in info.info) {
```

That reordering is to fix missing READMEs on code.dlang.org for packages with errors. It seems that most people simply keep ignoring errors anyhow.

---

Yeah it would have been good to check this when/before the package is registered ...

---

And possibly even earlier, e.g. maybe by integrating a lint function into dub.